frontend/src/__tests__/api.test.js

```diff
@@ -54,6 +54,94 @@ expect.extend({
     }
 });
 
+/**
+ * Seeding the database with the minimal set of API calls to create an assignment.
+ */
+function dataBaseSeed(api = { apiGET, apiPOST }) {
+    const { apiGET, apiPOST } = api;
+    let session = null;
+
+    beforeAll(async () => {
+        await apiPOST("/admin/debug/snapshot");
+        await apiPOST("/admin/debug/clear_data");
+    }, 15000);
+
+    //Create Session
+    const minimalSessionData = {
+        start_date: new Date("2020/02/10").toISOString(),
+        end_date: new Date("2020/12/31").toISOString(),
+        name: "Minimal Session Data",
+        rate1: 50
+    };
+
+    it("create a session", async () => {
+        const resp1 = await apiPOST("/admin/sessions", minimalSessionData);
+        const { payload: minimalSession } = await apiGET("/admin/sessions");
+        console.log("minimal session: \n", minimalSession);
+    });
+
+    //Create Position
+    const minimalPositionData = {
+        session_id: 1,
+        position_code: "CSC494",
+        position_title: "Capstone Project",
+        hours_per_assignment: 20,
+        start_date: "2020/01/01",
+        end_date: "2020/05/01"
+    };
+
+    it("create a position", async () => {
+        const resp1 = await apiPOST(
+            `/sessions/${minimalPositionData.session_id}/positions`,
+            minimalPositionData
+        );
+        const { payload: minimalPosition } = await apiGET(
+            `/sessions/${minimalPositionData.session_id}/positions`
+        );
+        console.log("minimal position: \n", minimalPosition);
+    });
+
+    //Create Applicant
+    const minimalApplicantData = {
+        utorid: "cole",
+        student_number: "10000000",
+        first_name: "Cole",
+        last_name: "Zemel",
+        email: "cole.zemel@gmail.com",
+        phone: "4166666666"
+    };
+
+    it("create an applicant", async () => {
+        const resp1 = await apiPOST(`/admin/applicants`, minimalApplicantData);
+        const { payload: minimalApplicant } = await apiGET(`/applicants`);
+        console.log("minimal applicant: \n", minimalApplicant);
+    });
+
+    //Create Assignment
+    const minimalAssignmentData = {
+        position_id: 1,
+        applicant_id: 1,
+        start_date: "2020/01/01",
+        end_date: "2020/05/01",
+        note: "N/A",
+        offer_override_pdf: "N/A",
+        active_offer_status: 1,
+        active_offer_id: 12345678
```

Comment:
??? You need to get every ID field from the database. No making them up!

Reply:
`active_offer_id` cannot be found in the database because there's no API call for Offer yet.